internal/cephfs/errors.go

```diff
@@ -33,6 +33,9 @@ var (
 	// ErrClonePending is returned when snapshot clone state is `pending`
 	ErrClonePending = errors.New("clone from snapshot is pending")
 
+	// ErrCloneIsEmpty is returned when the clone state is empty
+	ErrCloneIsEmpty = errors.New("clone is empty")
```

Comment:
clone state is empty? or an invalid clone state?

Reply:
Yep, an invalid clone state seems more appropriate. Will update the PR.